Assets/HoloToolkit/Input/Scripts/Focus/FocusManager.cs

```diff
@@ -355,18 +404,20 @@ public bool TryGetSinglePointer(out IPointingSource pointingSource)
         public delegate void PointerSpecificFocusChangedMethod(IPointingSource pointer, GameObject oldFocusedObject, GameObject newFocusedObject);
         public event PointerSpecificFocusChangedMethod PointerSpecificFocusChanged;
 
+        [Obsolete("Use either GetGazePointerEventData or GetSpecificPointerEventData")]
         public PointerInputEventData GetPointerEventData()
         {
-            if (UnityUIPointerEvent == null)
-            {
-                UnityUIPointerEvent = new PointerInputEventData(EventSystem.current);
-            }
-            else
-            {
-                UnityUIPointerEvent.Clear();
-            }
+            return GetGazePointerEventData();
+        }
+
+        public PointerInputEventData GetGazePointerEventData()
```

Comment:
Why do we need two different Get methods here? Won't `GetSpecificPointerEventData` return the Gaze data if you pass in the right pointer?

Reply:
Believe me I tried to make it that way but the `FocusEnter` and `FocusExit` methods don't supply a pointer source.

Reply:
Does that mean `FocusEnter` and `FocusExit` will only work in uGUI with gaze?

Reply:
It's unchanged and continues to work with both physics and graphics raycasters.